File: src/main/java/org/example/FizzBuzz.java
package org.example;

public class FizzBuzz {
    public static String execute(int number) {
        if (number == 3) {
            return "fizz";
        } else if (number == 5) {
            return "buzz";
        } else if (number == 15) {
            return "fizzbuzz";
        }

        return "1";
    }
}

File: src/test/scala/FizzBuzz.scala
// For more information on writing tests, see
// https://scalameta.org/munit/docs/getting-started.html
class FizzBuzz extends munit.FunSuite {
  test("The number 1 must be '1'") {
    assertEquals(fizzBuzz(1), "1")
  }

  test("The number 3 must be 'Fizz'") {
    assertEquals(fizzBuzz(3), "fizz")
  }
}
